src/ImageRunModal.jsx

```diff
@@ -851,7 +844,7 @@ export class ImageRunModal extends React.Component {
                                                       </TextList>
                                                   </TextContent>
                                                   <TextContent>
-                                                      <Text component={TextVariants.h4}>{cockpit.format("$0 $1", _("User:"), this.props.user)}</Text>
+                                                      <Text component={TextVariants.h4}>{_("User")}</Text>
```

Comment:
Let's revert this temporarily due to the soft string freeze. It makes sense conceptually to not talk about a *specific* user here, but (1) since the sorting of `this.props.users` it is easy to keep the old format, and (2) I'll change this back in the follow-up PR that actually introduces multi-user support.

Reply:
Done.